sky/backends/backend_utils.py

```diff
@@ -2047,11 +2050,10 @@ def get_node_counts_from_ray_status(
             require_outputs=True,
             separate_stderr=True)
         if rc:
-            raise RuntimeError(
-                f'Refreshing status ({cluster_name!r}): Failed to check '
-                f'ray cluster\'s healthiness with '
-                f'{instance_setup.RAY_STATUS_WITH_SKY_RAY_PORT_COMMAND}.\n'
-                f'-- stdout --\n{output}\n-- stderr --\n{stderr}')
+            raise exceptions.CommandError(
+                rc, instance_setup.RAY_STATUS_WITH_SKY_RAY_PORT_COMMAND,
+                f'Failed to check ray cluster\'s healthiness. \n'
+                f'-- stdout --\n{output}\n', stderr)
```

Comment:
```suggestion
                f'Failed to check ray cluster\'s healthiness.\n'
                '-- stdout --\n'
                f'{output}\n',
                stderr)
```
nit, just so code looks more similar to output